tests/AutoReview/ProjectCodeTest.php

```diff
@@ -500,6 +502,85 @@ static function ($className) {
         );
     }
 
+    /**
+     * @dataProvider provideAllCodeIsMonolithicPhpCases
+     *
+     * @param string $file
+     */
+    public function testAllCodeIsMonolithicPhp($file)
```

Comment:
I guess the name is to change.

Now it looks quite complicated - isn't the point of the test to see if a file contains only 1 classy element?

Can we just go from the file start and:
 - get next element of kind (class, interface or trait)
 - get next `{`
 - find block end
 - check that there is nothing after it

Wouldn't that be enough? That would work for PR mention as the reason for this test, right?

Reply:
great idea, I'll rewrite soon, thanks!